### ISSUE TITLE
Getting introduced to Activation Functions in Neural Networks

### ISSUE THREAD

**Issue by prathimacode-hub** [girlscript/winter-of-contributing#1524]
Welcome to 'DSWP' Team, good to see you here.

This issue will helps readers in acquiring all the knowledge to learn about getting involved with Activation Functions in Neural Networks. Tutorial to Activation Functions in Neural Networks  and how it's applied using sample code.

To get assigned to this issue, add your serial numbers mentioned in the spreadsheet of "Data Science with Python", the approach one would follow and choice you prefer (Documentation, Audio, Video). You can go with all three or any number of options you're interested to work on.

This issue is created for beginners and who aren't assigned to any issues yet.

If you had referred any resources, add them up in "DS Resources". Similarly if you had used datasets, include them in "DS Datasets".

Domain : Artificial Intelligence

Mentors Assigned : Gauri Bharadwaj / Jivitesh Jain

Points to Note :

The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.

"Issue Title" and "PR Title should be the same. Include issue number along with it.

Changes should be made inside the Datascience_With_Python/ directory & Datascience_With_Python branch.

Follow Contributing Guidelines & Code of Conduct before start Contributing.

This issue is only for 'GWOC' contributors of 'DSWP' domain.

All the best. Enjoy your open source journey ahead. 😎

**Comment by ELLIPSIS009** [comment]
I would like to work on this
Abhinav Mathur- 573
Type: Documentation
Approach: what is activation function in NN, example, types of activation functions, how it works, python implementation

**Comment by sriya26** [comment]
I would like to work on the documentation of this
Sriya Rallabandi-458
Approach: Introduction to various activation functions, mathematics behind it, Code required for the same.

**Comment by ELLIPSIS009** [comment]
@sriya26 as per the guidelines, The issues will be assigned on a first come first serve basis, so technically i am working on the documentation you can work on either video or audio.

**Comment by prathimacode-hub** [comment]
@ELLIPSIS009, Sorry to say but this issue is created on contributors request as per their mention in given thread and that too if it's responded in below 4 hours other person in queue is considered. So it would be her say first. But no worries you can take audio or video. FCFS basis comes only when issues are created on my own. 

Issue is assigned to @sriya26 for documentation.

**Comment by ELLIPSIS009** [comment]
ohh well i didnt know that it was self assign so i waited for your approval

**Comment by prathimacode-hub** [comment]
> ohh well i didnt know that it was self assign so i waited for your approval

No, you have to wait for approval but she has already addded up in thread about this issue to get it assigned for her so it's self assigned by her. @sriya26

**Comment by kjpavs** [comment]
I would like to contribute to this issue
Name: K J Pavithra
Batch: 7
Type: Video (ppt with audio)
Approach: explanation, different activation functions and a use case

**Comment by prathimacode-hub** [comment]
Issue assigned to @kjpavs for video contribution

**Comment by ashish-reddy-20-08** [comment]
@prathimacode-hub ,
I am M. Ashish Reddy. I would like to do AUDIO on this issue.
Batch number: 5
serial number:174.
Approach: A detailed explanation with examples and real life uses

**Comment by prathimacode-hub** [comment]
Issue assigned to @ashish-reddy-20-08 for audio contribution
